cocos/platform/ohos/CCApplication-ohos.cpp

```diff
@@ -0,0 +1,114 @@
+#if CC_TARGET_PLATFORM == CC_PLATFORM_OHOS
```

Comment:
Why do we need this macro, this file should be specific to openharmony?

Reply:
CCApplication-android.h has the similar macro, we are just trying to maintain consistency in style